bgpd/bgp_route.c

```diff
@@ -6955,6 +6955,7 @@ void route_vty_out(struct vty *vty, struct prefix *p,
 	json_object *json_nexthops = NULL;
 	json_object *json_nexthop_global = NULL;
 	json_object *json_nexthop_ll = NULL;
+	json_object *json_ext_community = NULL;
```

Comment:
Code looks good. However I think we should update the header of this command with appropriate information about ecom.

Currently header appears as below.

```
5349db61b3a6# show bgp l2vpn evpn route 
BGP table version is 1, local router ID is 5.1.1.1
Status codes: s suppressed, d damped, h history, * valid, > best, i - internal
Origin codes: i - IGP, e - EGP, ? - incomplete
EVPN type-2 prefix: [2]:[EthTag]:[MAClen]:[MAC]:[IPlen]:[IP]
EVPN type-3 prefix: [3]:[EthTag]:[IPlen]:[OrigIP]
EVPN type-4 prefix: [4]:[ESI]:[IPlen]:[OrigIP]
EVPN type-5 prefix: [5]:[EthTag]:[IPlen]:[IP]

   Network          Next Hop            Metric LocPrf Weight Path
Route Distinguisher: 5.1.1.1:2
*> [5]:[0]:[32]:[70.1.1.29]
                    110.1.26.1               0             0 101 i
Route Distinguisher: 5.1.1.1:3
*> [5]:[0]:[32]:[70.1.1.7]
                    110.1.29.1               0             0 101 i
```